egasub/submission/submit.py

```diff
@@ -54,39 +54,76 @@ def metadata_parser(ctx, metadata):
                     yaml_sample.get('bioSampleId'),
                     yaml_sample.get('sampleAge'),
                     yaml_sample.get('sampleDetail'),
-                    []
+                    [],
+                    None
         )
     
     run = Run(None,yaml_run.get('sampleId'),
               yaml_run.get('runFileTypeId'),
               yaml_run.get('experimentId'),
-              files)
+              files,None)
     
     return experiment, sample, run
 
 
 def perform_submission(ctx, submission_dirs):
+    echo("Login attempt with credentials in .egasub/config.yaml")
     login(ctx)
+    echo("Login success")
     submission = Submission('title', 'a description',SubmissionSubsetData.create_empty())
     prepare_submission(ctx, submission)
+    
+    experiment_ids = []
+    run_ids = []
+    sample_ids = []
 
     for submission_dir in submission_dirs:
+        echo("-------")
+        echo("Processing "+submission_dir)
+        
+        echo("Data parsing from experiment.yaml")
         experiment, sample, run = metadata_parser(ctx,os.path.join(ctx.obj['CURRENT_DIR'],submission_dir,'experiment.yaml'))
         
+        echo(" - Submission of the sample")
+        if not os.path.isdir(sample_log_directory(ctx,submission_dir)):
+            os.mkdir(sample_log_directory(ctx,submission_dir))
+        if not os.path.exists(sample_status_file(ctx,submission_dir)):
+            set_sample_status(ctx,submission_dir,"DRAFT")
+        if get_sample_status(ctx,submission_dir)=="VALIDATED":
+            continue
```

Comment:
At this moment, we can't skip the rest of the code yet. We still need to collect sample_id, experiment_id, run_id for the submission step.